casper/src/main/rholang/Either.rho

```diff
@@ -37,13 +46,20 @@ new rl(`rho:registry:lookup`), ListOpsCh in {
     //  r2 <- f2(r1)
     //  ...
     //} yield fn
-    contract @("Either", "compose")(@input, @functions, return) = {
+    contract Either(@"compose", @input, @functions, return) = {
       new combinator in {
         contract combinator(@head, @accumulatedValue, return) = {
-          @("Either", "flatMap")!(accumulatedValue, head, *return)
+          Either!("flatMap", accumulatedValue, head, *return)
         } |
         @ListOps!("fold", functions, ("Right", input), *combinator, *return)
       }
     }
-  }
+  } |
+  
+  rs!(
+    "89a6d9c47f360e8ce145f8fe3c773786dc86bd0e70d19643d02b0eb126473c55".hexToBytes(), 
+    (9223372036854775807, bundle+{*Either}), 
+    "29c68a1d8e792eeaf88158cfcb9a6038ebdfea81c7193025c41f3f3b6731e53c3d593b8d773df11caee97d5933623eb9fa60992cab75b770cacfb873f6182c0a".hexToBytes(), 
+    *uriOut
```

Comment:
Are we leaving `Nil` as an allowed forgable name to send on? I thought the plan was to disallow sending on any forgable name.

Reply:
It's not that they'll be disallowed, it's that sends and receives on forgeable names get discarded immediately in the tuplespace.  So there's a record in the block that something happened there, but it can't be used to trigger on-chain computation.